cart-service/scripts/fillDataBase.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { Client } = require('pg');
+require('dotenv').config();
```

Comment:
IMO it is a good practice to create a `.env.example` file to be able to push required env vars in your `.env` file BUT DO NOT USE REAL CREDENTIALS! 

Example content of `.env.example`
```
DB_HOST={db_host}
DB_PORT={db_port}
....
```